Projects/l0/Potentiometer/lib/Potentiometer/potentiometer.c

```diff
@@ -123,8 +58,11 @@ static void Potentiometer_MsgHandler(service_t *service, msg_t *msg)
         msg_t pub_msg;
         // fill the message infos
         pub_msg.header.target_mode = ID;
-        pub_msg.header.target      = msg->header.source;
+        // the message destination is the service that asked information
```

Comment:
small improve : send value of potentiommetre only when angle is differente from prev value